Desktop view queries

aceDavon opened the pull request.

aceDavon committed:
Write media queries for desktop view

With mobile first approach, write media queries to handle breakpoints
from 768px(Desktop) upwards displaying the desktop version of the template

aceDavon committed:
Create queries for footer links

aceDavon committed:
Fix linter errors on index.html

zieeco commented on css/style.css:
```diff
@@ -251,8 +263,7 @@ main {
 
 .grid-section hr,
```
- Please, kindly ensure the lines are having the correct color of #646369 as used in figma to match the design as yours is not as of right now.

:point_right: *This is applicable to all the lines in your design*

zieeco replied:
- [ ] Please, kindly ensure that your `language`, `framework`, and `skills` are well laid out to match the Figma design of the mobile version as yours are all placed horizontally below:

![span](https://user-images.githubusercontent.com/59397159/177810046-251cb5dc-df78-4f1a-ae67-5ff219aa1635.png)

*yours*
![here](https://user-images.githubusercontent.com/59397159/177810534-bb619393-e7f9-4c47-b111-91873b04a2e5.png)

zieeco replied:
*README*

- Please, kindly ensure your MIT.md link is not having a broken link but takes users to the licensed content.

zieeco replied:
*FORM BUTTON*

- [ ] Please, kindly update the text content of the form button to `send` as this is a form and users should make submissions to your form and not see the project when they make submissions to the form to improve the user experience and also, to match the Figma design.

zieeco reviewed (changes requested):
Hi @aceDavon,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


<img src="https://media0.giphy.com/media/8UGVQESpDMMQl7yd3F/giphy.gif"/>


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

aceDavon committed:
Fix issues raised by reviewres on index.html

tresorsawasawa commented on index.html:
```diff
@@ -150,7 +156,7 @@ <h1 class="contact-header">Contact Us</h1>
               <li><input type="text" name="fullname" required maxlength="30" placeholder="Text Input" /></li>
               <li><input type="email" name="email" required placeholder="debmirmij@gmail.com" /></li>
               <li><textarea name="message" required maxlength="500" placeholder="Enter text Here"></textarea></li>
-              <li><button class="btn btn-alt" type="submit">See Project</button></li>
+              <li><button class="btn btn-alt" type="submit">Send</button></li>
             </ul>
           </form>
         </div>
```
- [ **OPTIONAL** ] I think it would be better if you add the horizontal line between the `form` and the `footer` to stick to the Figma design.

- [ ] Good job by making some of the changes required by the previous reviewer. But I noticed that you missed one required change from your README file. Your `[MIT](./MIT.md)` link is broken, when it is clicked it displays an error page. Make sure you also fix this and your project will be completed. **You can do it by only changing this(in README file):** ``[MIT](./MIT.md)`` to `[MIT](./LICENSE)` as you already have the License file or another solution rename the `LICENSE` file to `MIT.md` and your problem will be fixed.

tresorsawasawa reviewed (changes requested):
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @aceDavon 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you add a short description on your pull request telling in detail what changes are inside.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

aceDavon committed:
Fix Issues raised from reviews

Create a MIT.md file to fix broken link on README.md,
Add extra dividers to enhance the look of the interface

aceDavon committed:
Fix errors from linters

Nemwel-Boniface reviewed (approved):
## Hello @aceDavon      :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes that were requested by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 

![](https://media.giphy.com/media/3kuSo744UIPJjcJUEn/giphy.gif)

## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 
## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
